satviz-consumer/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     java
+    `java-library`
```

Comment:
Wieso das?

Reply:
Ich brauch java.awt für die Desktop-Klasse in VisualizationController::screenshot. Ich wusste nich, wo ich das sonst hernehmen soll

Reply:
Dieses Plugin hat keinen Einfluss auf die Bibliotheken, die du zur Verfügung hast. java.awt gibt es auch so

Reply:
Okay, irgendwie hatte ich den Eindruck, dass erst damit java.awt gefunden wird. Hab's jetzt mal rausgenommen und es wird immer noch gefunden 👍